scripts/tests-integration.sh

```diff
@@ -18,7 +18,6 @@ SCRIPT_PATH=`cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd`
 cd ${SCRIPT_PATH}/..
 
 # TESTS SUITE
-./scripts/tests-integration-stage.sh --stage=DUMMY
```

Comment:
Not needed anymore?

Reply:
Not really, the other tests do more than this one.